prowler/providers/common/audit_info.py

```diff
@@ -287,3 +292,26 @@ def set_provider_audit_info(provider: str, arguments: dict):
         sys.exit()
     else:
         return provider_audit_info
+
+
+def get_tagged_resources(input_scan_tags: list, current_audit_info: AWS_Audit_Info):
+    """
+    get_tagged_resources get the resources that have the tags input by the user and are going to be scanned
+    """
+    scan_tags = []
+    tagged_resources = []
+    if input_scan_tags:
+        for tag in input_scan_tags:
+            key = tag.split("=")[0]
+            value = tag.split("=")[1]
+            scan_tags.append({"Key": key, "Values": [value]})
+        # Get Resources with scan_tags for all regions
+        for region in current_audit_info.audited_regions:
+            client = current_audit_info.audit_session.client(
+                "resourcegroupstaggingapi", region_name=region
+            )
+            get_resources_paginator = client.get_paginator("get_resources")
+            for page in get_resources_paginator.paginate(TagFilters=scan_tags):
+                for resource in page["ResourceTagMappingList"]:
+                    tagged_resources.append(resource["ResourceARN"])
+    return tagged_resources
```

Comment:
Include a `try/catch` block here and handle the possible errors. We need to include it in each function.

Reply:
Done